assignments/arrays.js

```diff
@@ -63,32 +63,56 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car ;
+for (i = 0; i < inventory.length; i++) {
+    if(inventory[i].id === 33){
+        car = inventory[i]
+    }
+}
+
+console.log(`Car 33 is a ` + car.car_year + " " + car.car_make + " " + car.car_model);
```

Comment:
Excellent work-around and use of concatenation. Here's the quickest answer using interpolation:
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );